eng/common/scripts/Cadl-Project-Generate.ps1

```diff
@@ -4,7 +4,9 @@
 param (
     [Parameter(Position=0)]
     [ValidateNotNullOrEmpty()]
-    [string] $ProjectDirectory
+    [string] $ProjectDirectory,
+    [Parameter(Position=1)]
+    [string] $CadlAdditionalOptions ## addtional cadl emitter options, separated by semicolon if more than one
```

Comment:
Can we modify this to accept just `key=value;key=value` and not require the user to pass in the emitter name?  We know this from `$emitterName` and can construct this part for the customer to simplify their command line call.

Reply:
updated